### PR TITLE
change paths in update-dapps.yml and scripts/update-dapps

### PR DESCRIPTION
**Pull request by LXIF**
change paths in update-dapps.yml and scripts/update-dapps to new icon location that allows dynamic loading and caching

<hr/><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

File: .github/workflows/update-dapps.yml [hunk 1]
```diff
@@ -32,13 +32,14 @@ jobs:
           base: main
           add-paths: |
             src/frontend/src/flows/dappsExplorer/dapps.json
-            src/frontend/assets/icons
+            src/frontend/assets/src/assets/icons
```
Comment (lmuntaner) [inline .github/workflows/update-dapps.yml:35]: Ditto, is this correct? I was expecting it to be `frontend/src/assets/icons`

Could you try running this workflow and see that it works as expected?

Comment (LXIF) [reply .github/workflows/update-dapps.yml]: @lmuntaner happy to do that, need to check how to do that though

Comment (lmuntaner) [reply .github/workflows/update-dapps.yml]: I believe that it's also better to introduce this change along with the PR that introduces caching.

Comment (LXIF) [reply .github/workflows/update-dapps.yml]: Yeah makes sense at the end of the day

**Comment by lmuntaner** [comment]
Closing because it's added in #2697

**Pull request closed without merging**